### DISCUSSION TITLE
RSE and the academia - industry divide (Session 1 - Main room / Session 2 - Room 1))

### DISCUSSION THREAD

**Discussion by klapaukh** [RSEAA/RSEAA.github.io#135] [Discussion topics for #RSEAA24]
A lot of the time we focus on RSE within the university / public research setting. These institutions have their own particular hierarchies, ways of working, and funding mechanisms. However, many RSE roles will overlap with industry jobs, for example:
 - industrial R&D has research
 - start-ups have staff needing to do little bits of everything
 - industry also has to release software to users

Should we, as an RSE community, be seeking out more connections with industry to help with things like:
 - Helping move RSEs with unstable academic positions to see options in more stable industrial (perhaps even industrial research) roles?
 - Helping RSEs who are passionate about a tool or product turn it into a startup? 
 - Bringing industrial researchers closer to academic researchers? 
 - And more.

**Comment by rowlandm** [comment]
I want to reach out to startups.

**Comment by rowlandm** [comment]
What about spinning out a RSE company? I know people doing software consulting but not as a group.

**Comment by rowlandm** [comment]
People in industry [use Mixomics](https://github.com/mixOmicsTeam/mixOmics)

**Comment by ikfj** [comment]
I'm from National Institute of Informatics, Japan.
It is common for universities and institutes in Japan to order development of research software to software vendors. RSEs are active in such vendors, often from small and medium-sized companies. From an academia perspective, the problem is that good vendors are hard to find. Large contractors often win large bids for large projects, but the quality of the finished software tends to be inferior. We are interested in creating a mechanism to match universities with smaller software vendors with skilled developers.

**Reply by rowlandm** [reply]
Thanks @ikfj !

**Reply by hot007** [reply]
I was actually just thinking about the related issue where a lot of industry relies on proprietary software tools that we might tend to eschew in academia (due in part to costs!) which can lead to our RSEs having less experience with their tools and more experience in creating or working with open source tools. Of course, we want to avoid vendor lock-in and the rate of universities training students in these proprietary tools via 'Education' licenses that can similarly negatively impact on graduates' ability to transition into the open source domain.

**Comment by rowlandm** [comment]
[BioGrid is another company](https://www.biogrid.org.au/) that works with clinical trial data
